lesson05/script.js

```diff
@@ -0,0 +1,64 @@
+function startApp() {
+    let input = '';
+    let usersBalance = 1000;
+    const maxDepositCap = 50000;
+    let lowBalanceCap = 300;
+
+    do {
+
+        let thePrompt = prompt(`Welcome to the Bank. \n Press Q to Exit. \n Press W to Withdraw \n Press D to Deposit. \n Press B to View Blance`);
+        input = thePrompt;
+        let withrdawPrompt = '';
+        let depositPrompt = '';
+
+        switch (input) {
```

Comment:
My only suggestion here would be to use `input.toUpperCase()` to convert all input to capitals. So if a user enters a lowercase letter, it'll still open the correct option.